lib/auth/apiserver.go

```diff
@@ -1503,6 +1507,10 @@ func (s *APIServer) emitAuditEvent(auth ClientI, w http.ResponseWriter, r *http.
 	if err := auth.EmitAuditEvent(req.Type, req.Fields); err != nil {
 		return nil, trace.Wrap(err)
 	}
+	if req.Type == events.SessionStartEvent {
```

Comment:
this logic does not belong to api handlers, should be moved to the actual auth server